### ISSUE TITLE
Desktop background still covers the whole screen on startup

### ISSUE THREAD

**Issue by maxigaz** [avivace/dotfiles#7]
I've added your i3 config file and created `~/.config/plasma-workspace/env/wm.sh` based on yours, but the desktop background still covers the entire screen so that nothing else can be seen (except for the mouse cursor).

I've tried modifyng the following line towards the bottom in `~/.config/i3/config`
```
for_window [title="Desktop — Plasma"] kill; floating enable; border none
```
by replacing "Desktop" with its equivalent in my mother tongue (the language in Plasma is not set to English), but I have the same results. To identify the actual name of the "window" of plasma's background, I used clicking on a window's icon → More actions → Special window properties → tab Window match → Identify window properties → clicking somewhere on the desktop in a normal Plasma session.

Using Manjaro Linux with Plasma 5.12.

**Comment by subdavis** [comment]
It is incredible that we would have the same problem in such close temporal proximity.

I've been trying to figure this out for DAYS -- nobody seems to mention that you need to stop `/usr/bin/ksplashqml` from starting too.

In your bin directory, move `/usr/bin/ksplashqml` to `/usr/bin/ksplashqml.old` to prevent it from starting.  

@avivace could you mention this in the readme too?  Without doing this, the desktop is still covered with the splash loading screen for the first ~10-15 seconds of boot. 

I figured this out by profiling `ps -aux` before and after the splash exited.

**Comment by avivace** [comment]
Thank you, I'll add it to the instructions.

Interesting enough, you can have the "Plasma Desktop", with widgets and icons, as Workspace 1 if you just disable `ksplashqml` without killing Plasma (the `wmctrl -c Plasma` in i3 confinguration).

**Comment by maxigaz** [comment]
I've made some progress. According to my experience, renaming `/usr/bin/ksplashqml` is not necessary as long as the splash screen is disabled in System settings → Workspace Theme → Splash Screen. However, I also needed to comment the line that launches compton in `~/.config/plasma-workspace/env/wm.sh`, otherwise the only thing that becomes visible is the background set by `exec --no-startup-id xsetroot -solid "#333333"` in `~/.config/i3/config` (I put it there for testing purposes).

That out of the way, I still have some issues. (Maybe it should be in a different GitHub issue?)

- I can see the solid background for a few seconds, but after Plasma is loaded, the background turns black and the edge of the panel created in Plasma flickers.
- The panel is supposed to appear on the right side of the screen, but appears on the left instead.
- Also, if I launch an application like Firefox, its window remains invisible. I can still see the panel, Yakuake, and the logout prompt though.

I have an (old) AMD Radeon HD6670 using the free drivers, by the way.

**Comment by avivace** [comment]
Do you have any progress? Did you managed to obtain a decent startup? I tried setting this up in a Manjaro/Arch VM and I don't see any of these.
It's probably caused by compton/drivers at this point. What happens if you stop compton and set the background using `feh` ?

**Comment by maxigaz** [comment]
No, I haven't experimented with it any more since then. It's entirely possible that it's a driver issue, indeed. (Now and again I see graphical glitches when I resize windows or during startup, but luckily they are always temporary.)

> What happens if you stop compton and set the background using `feh` ?

The same thing I wrote: flickering, misplaced panel etc.. Or by stopping compton, do you mean explicitly killing it *while* it's running?

**Comment by FreddyJRC** [comment]
I have found a posible solution.
Edit the `~/.config/plasma-workspace/env/wm.sh` change the backend to `xrender` and delete the vsync

```
# Disable KWin and use i3gaps as WM
export KDEWM=/usr/bin/i3

# Compositor (Animations, Shadows, Transparency)
# xcompmgr -C
compton -cCFb --backend xrender
```
i have a HD 6750 so this could help you.
